### PR TITLE
Bring updates from Master to Preview

### PR DESCRIPTION
**Pull request by barustum**
## Checklist
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [ ] This pull-request is submitted against the `master` branch.

### CODE REVIEW

**Review by abhipsaMisra — APPROVED** [review]
The service API versions and csproj versions, all look good!

**Review by azabbasi — APPROVED** [review]

**Review by vinagesh — APPROVED** [review]

File: iothub/device/src/Common/Api/ClientApiVersionHelper.cs [hunk 1]
```diff
@@ -6,15 +6,12 @@ namespace Microsoft.Azure.Devices.Client
     internal class ClientApiVersionHelper
     {
         internal const string ApiVersionQueryPrefix = "api-version=";
-        internal const string ApiVersionLatest = "2019-07-01-preview";
 
-        // The preview API version has been added to enable support for plug and play features.
-        // This will be removed once the plug and play service goes GA.
-        internal const string ApiVersionPreview = "2020-05-31-preview";
+        // The preview branch uses a GA api version because Device Streaming is already in public preview and is available on certain Azure regions.
```
Comment (bikamani) [inline iothub/device/src/Common/Api/ClientApiVersionHelper.cs:10]: 👍

**Pull request merged**